test/server/admin/prometheus_stats_test.cc

```diff
@@ -418,6 +441,9 @@ envoy_cluster_test_4_upstream_cx_total{another_tag_name_4="another_tag_4-value"}
 # TYPE MYAPP_test_bar gauge
 MYAPP_test_bar{tag_name="tag-value"} 0
 
+# TYPE envoy_control_plane_identifier gauge
+envoy_control_plane_identifier{cluster="c1",text_value="CP-1"} 1
```

Comment:
yeah let's throw in a testcase where the value has a double-quote.

Reply:
Such a test case already exists [here](https://github.com/envoyproxy/envoy/blob/main/test/server/admin/prometheus_stats_test.cc#L147). Since escaping of characters is the responsibility of `PrometheusStatsFormatter::formattedTags` and it's already tested, I think it's not necessary to cover this case twice. What do you think?